README.md

```diff
@@ -18,22 +18,22 @@
 [OpenAPI](https://swagger.io/docs/specification/about/) specification is an API description format for REST APIs where you can describe your entire API, including:
 
 - Available endpoints (/users) and operations on each endpoint (GET /users, POST /users)
-- Operation parameters Input and output for each operation
+- Operations input and output parameters
 - Authentication methods
-- Contact information, license, terms of use and other information.
+- Contact information, license, terms of use and other information
 
 ## Aim
 
-Although there are a lot of advantages to use OpenAPI like documentation & code generation, publish your API faster, promote your teams collaboration, etc., industrialize it remains complicated due to certain shortcomings. Using a same OpenAPI file into several contexts is not supported by the current release of OpenAPI (to propose a same API but which could be used in public and private use). Using the external references to design your API from several files to facilitate the maintainability is still involved. Painlessly publish your specifications into a software repository to be able to reuse it (to generate client or server code). Etc.
-Also, the OpenAPI ecosystem is very large (https://openapi.tools/) with a lot of tools with different quality level. Consequently, to set up a complete development environment, many tools has to be installed and upgraded regularly.
+Although using OpenAPI comes with a lot of advantages (documentation and code generation, faster API publishing, teams collaboration promotion...), industrializing it remains complicated due to certain shortcomings. Using a single OpenAPI file for several contexts (public and private usage for example) is not supported by the current release of OpenAPI. Using the external references to design your API from several files to facilitate the maintainability is still required. Painlessly publish your specifications into a software repository to be able to reuse it (to generate client or server code). Etc.
```

Comment:
I don't really understand the meaning of the two last sentences here.

Reply:
I like your improvements :)